.github/ISSUE_TEMPLATE/bug_report_linux.yml

```diff
@@ -0,0 +1,73 @@
+name: Bug Report (Linux)
+description: File a bug report for Linux.
+labels: ["bug", "triage"]
+projects: ["freedomofpress/dangerzone"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Hi, and thanks for taking the time to open this bug report. 
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Can you specify the steps you followed, so that we can reproduce?
+      value: "A bug happened!"
+    validations:
+      required: true
+  - type: textarea
+    id: os-version
+    attributes:
+      label: operating system version
+      description: |
+        Which version of your operating sytem do you use?
+
+        On Linux, you should have good results with this command `cat /etc/os-release`
+    validations:
+      required: true
+  - type: textarea
+    id: dangerzone-version
+    attributes:
+      label: Dangerzone version
+      description: Which version of Dangerzone are you using?
+    validations:
+      required: true
+  - type: textarea
+    id: podman-info
+    attributes:
+      label: Podman info
+      description: |
+        If the bug occurs during document conversion, or is otherwise related
+        with Podman, please copy and paste the following commands in your
+        terminal, and provide us with the output:
```

Comment:
I guess we should turn this into a single line, and let GitHub wrap it where necessary.